proto/genesis/v1beta/types.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
+package spn.genesis.v1beta1;
+
+option go_package = "github.com/tendermint/spn/x/genesis/types";
+
+import "google/protobuf/any.proto";
+import "gogoproto/gogo.proto";
+
+message Chain {
+  string chainID = 1;
+  string creator = 2;
+  repeated string peers = 3; 
+  string sourceURL = 4; 
+  string sourceHash  = 5; 
+  int64 createdAt = 6;
+  bytes initialGenesis = 7;
```

Comment:
Should we have a `finalGenesis` as well?

Reply:
To me, finalizing a genesis means switching on a toggle that freezes the `CurrentGenesis`, so no changes can be applied to it.